src/Microsoft.Sbom.Common/ComplianceStandard/NTIAComplianceStandardEnforcer.cs

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Microsoft.Sbom.Common.ComplianceStandard.Enums;
+using Microsoft.Sbom.Common.Spdx30Entities;
+using Microsoft.Sbom.Common.Spdx30Entities.Enums;
+using Microsoft.Sbom.Contracts.Enums;
+using Microsoft.Sbom.JsonAsynchronousNodeKit.Exceptions;
+using Microsoft.Sbom.Parsers.Spdx30SbomParser.ComplianceStandard.Interfaces;
+
+namespace Microsoft.Sbom.Common.ComplianceStandard;
+
+public class NTIAComplianceStandardEnforcer : IComplianceStandardEnforcer
+{
+    private static readonly IReadOnlyCollection<string> EntitiesWithDifferentNTIARequirements = new List<string>
+    {
+        "SpdxDocument",
+        "File",
+    };
+
+    public ComplianceStandardType ComplianceStandard => ComplianceStandardType.None;
+
+    public NTIAComplianceStandardEnforcer()
+    {
+    }
+
+    public string GetComplianceStandardEntityType(string? entityType)
+    {
+        if (EntitiesWithDifferentNTIARequirements.Contains(entityType))
+        {
+            return string.IsNullOrEmpty(entityType) ? string.Empty : "NTIA" + ComplianceExtensions.GetCommonEntityType(entityType);
```

Comment:
Use extension method:
```suggestion
            return string.IsNullOrEmpty(entityType) ? string.Empty : "NTIA" + entityType.GetCommonEntityType();
```